planner/core/integration_test.go

```diff
@@ -1575,6 +1575,22 @@ func (s *testIntegrationSuite) TestIndexHintWarning(c *C) {
 	}
 }
 
+func (s *testIntegrationSuite) TestIssue28052(c *C) {
```

Comment:
Why do you add this test under `planner/core` package?

Reply:
Oh, Sorry to put it in wrong place